status/status.go

```diff
@@ -152,7 +152,7 @@ func (s *Status) generateTestReport(ctx context.Context, detail intgteststat.Int
 }
 
 // generateText generates a text with details for the given state
-func (s *Status) generateText(ctx context.Context, integrationTestStatusDetail intgteststat.IntegrationTestStatusDetail, namespace string) (string, error) {
+func (s *Status) generateText(ctx context.Context, integrationTestStatusDetail intgteststat.IntegrationTestStatusDetail, namespace string, application *applicationapiv1alpha1.Application) (string, error) {
```

Comment:
It looks like the `application` parameter is unused in this function.

Reply:
Correct, thanks for catching that. Updated.